src/atoms/visual-story/web-story-page-components/web-story-page-components.tsx

```diff
@@ -24,7 +30,8 @@ const WebStoryPageComponentsBase = ({ card, config, story }: WebStoryPageCompone
 
   return (
     <Fragment>
-      {imageElement && (
+      {videoUrl && <VideoWebStory videoUrl={videoUrl} config={config} imageElement={imageElement} />}
+      {imageElement && !videoElement && (
```

Comment:
Also add the `!videoElement` condition below > `{(titleElement || textElements.length || imageElement)`
we don't want to show title/text elements on top of video
But CTA button should show

Reply:
done